book/src/06_system_data.md

```diff
@@ -18,8 +18,8 @@ impl<'a> System<'a> for Sys {
 }
 ```
 
-It is very important that you don't fetch both a `ReadStorage` and a `WriteStorage`
-for the same component or a `Fetch` and a `FetchMut` for the same resource.
+It is very important that you don't Read both a `ReadStorage` and a `WriteStorage`
```

Comment:
First of all read should either be de-capitalized or made code.
After that the sentence is super awkward: It's implying that there is reading of a `WriteStorage`.
Not sure how to fix it.